routers/api/v1/repo/milestone.go

```diff
@@ -32,10 +32,14 @@ func ListMilestones(ctx *context.APIContext) {
 	//   description: name of the repo
 	//   type: string
 	//   required: true
+	// - name: state
+	//   in: query
+	//   description: Milestone state, Recognised values are open, close and all. Defaults to "open"
```

Comment:
```suggestion
	//   description: Milestone state, Recognised values are open, closed and all. Defaults to "open"
```

I think the values Recognised are `open`, [`closed`](https://github.com/go-gitea/gitea/blob/ebe21b1457df56e8fdc19be47e086f6c9e429fdf/modules/structs/issue.go#L18) and `all` :+1: